### PR TITLE
[ErrorCatcher] Fixed some escaping in error renderers

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | -
| License       | MIT
| Doc PR        | not needed

Fixes this: https://github.com/symfony/symfony/pull/32364/files#r300394620

### CODE REVIEW

File: src/Symfony/Component/ErrorCatcher/ErrorRenderer/XmlErrorRenderer.php [hunk 1]
```diff
@@ -65,7 +66,7 @@ public function render(FlattenException $exception): string
         return <<<EOF
 <?xml version="1.0" encoding="{$this->charset}" ?>
 <problem xmlns="urn:ietf:rfc:7807">
-    <title>{$exception->getTitle()}</title>
+    <title>{$title}</title>
     <status>{$exception->getStatusCode()}</status>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/ErrorCatcher/ErrorRenderer/XmlErrorRenderer.php]: let's do the same for the status code and the charset, better be extra safe
maybe also check that this is done in other formats?

**Comment by yceruto** [comment]
Javier, could you please fix that for `HtmlErrorRenderer` too?

File: src/Symfony/Component/ErrorCatcher/ErrorRenderer/HtmlErrorRenderer.php [hunk 2]
```diff
@@ -94,11 +94,14 @@ public function setFileLinkFormat($fileLinkFormat)
      */
     public function getBody(FlattenException $exception)
     {
+        $statusCode = $this->escapeHtml($exception->getStatusCode());
+        $title = $this->escapeHtml($exception->getTitle());
```
Comment (yceruto) [inline src/Symfony/Component/ErrorCatcher/ErrorRenderer/HtmlErrorRenderer.php]: See `render` method too ;)

**Review by yceruto — APPROVED** [review]

**Review by Tobion — APPROVED** [review]

**Comment by yceruto** [comment]
Thank you @javiereguiluz.

**Pull request merged**